app/src/main/java/ru/startandroid/organizer/HomeActivity.java

```diff
@@ -1,14 +1,20 @@
 package ru.startandroid.organizer;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentTransaction;
 
 import android.os.Bundle;
 
 public class HomeActivity extends AppCompatActivity {
 
+    FragmentTransaction transaction;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_home);
+        transaction = getSupportFragmentManager ().beginTransaction ();
```

Comment:
Let's add savedInstanceState null checking. We usually add fragment only if savedInstanceState == null (first start of Activity). System will recreate this fragment all next restarts (rotate screen for example).